assets/src/edit-story/components/panels/design/pageAttachment/pageAttachment.js

```diff
@@ -136,18 +133,19 @@ function PageAttachmentPanel() {
         </Row>
       )}
 
-      {Boolean(url) && !isInvalidUrl && (
+      {showTextInput && (
         <Row>
-          <ExpandedTextInput
-            onChange={(value) => {
+          <Input
+            onChange={({ target }) => {
+              const { value } = target;
               // This allows smooth input value change without any lag.
               _setCtaText(value);
               debouncedCTAUpdate(value);
             }}
-            onBlur={(atts = {}) => {
-              const { onClear } = atts;
-              if (onClear) {
+            onBlur={({ target }) => {
+              if (!target.value) {
                 updatePageAttachment({ ctaText: defaultCTA });
+                _setCtaText(defaultCTA);
               } else {
                 cancelCTAUpdate();
                 updatePageAttachment({
```

Comment:
💭 should we move this to its own event handler for better readability? Seems a little beefy for an inline function.